_posts/2021-04-28-hhvm-4.107.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: "HHVM 4.106"
+layout: post
+author: fred 
+category: blog
+---
+
+HHVM 4.107 is released! This release marks the end of support for 4.101;
+HHVM 4.102&ndash;4.106 remain supported, as does the 4.80 LTS release.
+
+# Highlights
+
+- Added an hhconfig option to ban partially abstract type constants; we expect the
+  `disallow_partially_abstract_typeconst_definitions` flag to be on by default
+  in a future release.
```

Comment:
Not to be confused with `disable_partially_abstract_typeconsts`. This is a different option, with a very similar name. It might be worth disambiguating.